test/js/node/worker_threads/worker_threads.test.ts

```diff
@@ -20,6 +20,18 @@ import wt, {
   workerData,
 } from "worker_threads";
 
+// shorthand to set environment data while ensuring that it will not persist beyond the current test
+function setEnvironmentDataInScope(key: string, data: wt.Serializable): Disposable {
+  setEnvironmentData(key, data);
+  return {
+    [Symbol.dispose]() {
+      // delete the key. ignored because @types/node does not think you can pass undefined to setEnvironmentData
+      // @ts-ignore
```

Comment:
Use // @ts-expect-error instead

The difference is that ts-expect-error _will_ error itself if the line starts working again in the future. This means we know to safely remove it in the future

Reply:
Ha, that's a user on GitHub 😆